threestudio/models/geometry/implicit_volume.py

```diff
@@ -12,12 +12,14 @@
     contract_to_unisphere,
 )
 from threestudio.models.networks import get_encoding, get_mlp
+from threestudio.utils.base import Updateable
+from threestudio.utils.misc import C
 from threestudio.utils.ops import get_activation
 from threestudio.utils.typing import *
 
 
 @threestudio.register("implicit-volume")
-class ImplicitVolume(BaseImplicitGeometry):
+class ImplicitVolume(BaseImplicitGeometry, Updateable):
```

Comment:
This is not needed as BaseImplicitGeometry (inherit from BaseModule) is already Updateable.